sympy/core/expr.py

```diff
@@ -23,6 +23,11 @@ class Expr(Basic, EvalfMixin):
     used only for argument storage and expression manipulation, i.e.
     pattern matching, substitutions, etc).
 
+    If you want to override the comparisons of expressions:
+    Should use _eval_is_ge for inequality, or _eval_Eq, with multiple dispatch.
+    _eval_is_ge return true if x >= y, false if x < y, and None if the two types
+    cannot be compared.
```

Comment:
This should say `_eval_is_eq` rather than `_eval_Eq`. Also "if the two types are not comparable or the comparison is indeterminate".